### ISSUE TITLE
Gene ID not populating for certain GSE's

### ISSUE THREAD

**Issue by snoreis** [ctlab/phantasus#155]
It seems like for certain GEO data sets the gene id and gene symbol do not populate.  For example (screenshot attached) GSE110420 only populates an ID field, while other GEO databases (GSE13432, for example) produce a gene id and gene symbol just fine.  

Any thoughts? 

<img width="1168" alt="image" src="https://user-images.githubusercontent.com/66849990/84548149-9ddb7d00-acca-11ea-836e-280d53278c37.png">

**Comment by assaron** [comment]
For this particular dataset the gene annotation present in GEO is not that good. At the moment we're thinking how to better handle such datasets.

**Comment by snoreis** [comment]
Thanks for responding.

I eventually got around it by downloading and modifying the GSE file.  Fortunately the information in the associated GPL file is easy to extract, so I amended the gene id information to the GSE file.

<img width="1438" alt="image" src="https://user-images.githubusercontent.com/66849990/84667751-60ecd180-aee8-11ea-9526-9a1c13519c33.png">

Perhaps there could be a second drop-down box where one enters the GPL file number?  Or someone could just include a short tutorial about how to add GPL information to the GSE file?
